File: src/main/java/ua/nure/fedorenko/kidstim/model/entity/Child.java
package ua.nure.fedorenko.kidstim.model.entity;

import javax.persistence.Column;
import javax.persistence.Entity;
import javax.persistence.Table;
import java.io.Serializable;

@Entity
@Table(name = "child")
public class Child extends User implements Serializable {


    @Column
    private int gender;

    @Column
    private int points;

    @Column
    private long birth;

    public long getBirth() {
        return birth;
    }

    public void setBirth(long birth) {
        this.birth = birth;
    }

    public int getGender() {
        return gender;
    }

    public void setGender(int gender) {
        this.gender = gender;
    }

    public int getPoints() {
        return points;
    }

    public void setPoints(int points) {
        this.points = points;
    }
}

File: src/main/java/ua/nure/fedorenko/kidstim/model/entity/Child.kt
package ua.nure.fedorenko.kidstim.model.entity

import org.hibernate.annotations.GenericGenerator
import java.io.Serializable
import javax.persistence.*

@Entity
@Table(name = "child")
data class Child(
        @Id
        @GeneratedValue(generator = "uuid")
        @GenericGenerator(name = "uuid", strategy = "uuid")
        @Column(name = "id")
        var id: String,
        @Column
        var email: String,
        @Column
        var name: String? = null,
        @Column
        var surname: String? = null
) : Serializable {

    @Column
    var password: String? = null

    @Column
    var photo: String? = null

    @Column(name = "device_token")
    var deviceToken: String? = null
    @Column
    var gender: Int = 0

    @Column
    var points: Int = 0

    @Column
    var birth: Long = 0
}